src/main/java/com/cmlteam/serv/HttpHandlerServeFilesTar.java

```diff
@@ -0,0 +1,37 @@
+package com.cmlteam.serv;
+
+import com.sun.net.httpserver.HttpExchange;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.Set;
+
+class HttpHandlerServeFilesTar extends HttpHandlerBase {
```

Comment:
I have a feeling that now since we added this class - we can remove the `HttpHandlerServeFolderTar` class since now we can achieve the same functionality by this new class by providing `folder.listFiles()` as input argument to constructor. Could you please check and if this works - do it. The less code we have - better for support.

Reply:
resolved